stories/utils/generate-gtm-analytics-property-story.js

```diff
@@ -48,9 +48,9 @@ export function generateGTMAnalyticsPropertyStory( {
 	setUp = () => {},
 } ) {
 	return () => {
-		setUp();
-
 		const setupRegistry = ( registry ) => {
+			setUp( registry );
```

Comment:
While resolving a merge conflict which was around this logic, I moved the `setUp` function in here so that it can run logic on the registry, which was needed (`registerModule` is registry-specific, while the previous filter-based mechanism wasn't).